experiments/optimizer/README.md

```diff
@@ -0,0 +1,258 @@
+# Using the Dispatcher Optimizer Tool (DOT)
+
+This experiment demonstrates how to use the Dispatcher Optimizer Tool, its Maven Plug-In in particular, to create
+better configurations, following best practices and easily spotting inadvertent configuration errors.
+
+## Compatibility
+
+This experiment is compatible with both AEM 6.5 and AEM as a Cloud Service.
+
+## Problem
+
+The Dispatcher configuration is complicated and is easily left to the end of a project.  At that point it is
+tempting to do whatever it takes to get the Dispatcher to work without following steps that will avoid problems in
+the future.  The approach of "good enough" is often taken and only later after CSOs, privacy leaks and DoS attacks
+is the configuration updated.
+
+It is also easy to end up with a configuration with syntax errors.  These errors can be hard to spot when
+the dispatcher configuration is large.
+
+The DOT can be used a few ways:
+* Settings can be added to a Maven POM for the Dispatcher configuration.  This will trigger the
+  DOT to execute within the 'analyze' livecycle during development, giving the developer immediate feedback on the
+  perceived accuracy of the Dispatcher configuration.
+* The DOT is used in Adobe's AEM Cloud Service Pipeline, where it is run automatically to analyze the contained
+  configuration.
+* A developer can use the open-source DOT Core library to create a new application, perhaps for the command line.
+
+## Setup
+
+Follow the instructions on the main experiments page.
+
+***
+![#ff0101](https://via.placeholder.com/15/ff0101/000000?text=+)
```

Comment:
Just checking if this placeholder url is ok to be here?

Reply:
Ya?  It is a trick to add color to the README.  Kinda weird though.